dotnetv3/Polly/SynthesizeSpeechExample/SynthesizeSpeech.cs

```diff
@@ -1,32 +1,30 @@
-﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier:  Apache-2.0
 
-using Amazon;
-using Amazon.Polly;
-using Amazon.Polly.Model;
-using System.IO;
-using System.Threading.Tasks;
-
-namespace SynthesizeSpeech
+/// <summary>
+/// This example uses the Amazon Polly service to convert text to
+/// speech. It then saves the converted text to an MP3 file. The
+/// code was written against the AWS SDK for .NET 3.5 and .NET 5.
```

Comment:
This example uses Amazon Polly to convert text to speech. Then, it saves the converted text to an MP3 file. This code was written using the AWS SDK for .NET 3.5 and .NET 5.